packages/wallets/wallet-strategy/src/strategy/index.ts

```
@@ -1 +1,180 @@
+      })
+    default:
+      return undefined
+  }
+}
+
+const createAllStrategies = (
+  args: WalletStrategyArguments,
+): ConcreteStrategiesArg => {
+  return Object.values(Wallet).reduce(
+    (strategies, wallet) => ({
+      ...strategies,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)